src/app/GitUI/CommandsDialogs/FormVerify.cs

```diff
@@ -518,5 +587,10 @@ private void saveAsToolStripMenuItem_Click(object sender, EventArgs e)
                 }
             }
         }
+
+        protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
+        {
+            return mnuLostObjects.PreProcessMessage(ref msg) || base.ProcessCmdKey(ref msg, keyData);
```

Comment:
Please add a comment why this is needed.
Doesn't have `Form` an option like `KeyPreview`?

Reply:
Not an expert on this but your comment made me reevaluate.
I made some changes...